CoreScriptsRoot/Modules/Settings/Pages/GameSettings.lua

```diff
@@ -158,12 +158,16 @@ local function Initialize()
       this.PerformanceStatsMode,
       this.PerformanceStatsOverrideText = nil
 
-      local startIndex = 2
-
-      if GameSettings.PerformanceStatsVisible then
-        startIndex = 1
+      function GetDesiredPerformanceStatsIndex()
+        if GameSettings.PerformanceStatsVisible then
+          return 1
+        else
+          return 2
```

Comment:
does returning 2 here just make it so there are no stats available to show? wouldn't it be easier/more performant to just set Visible to false?